util/regexp-assemble/tests/preprocessor_test.py

```diff
@@ -631,3 +631,55 @@ def test_includes_content(self, context, include_file_name, include_file):
         output = assembler._run(Peekerator(contents.splitlines()))
 
         assert output == '(?:file contents|next line)'
+
+    def test_includes_content_recursive(self, context):
+        name1 = f'{uuid.uuid4()}.data'
+        name2 = f'{uuid.uuid4()}.data'
+
+        file1_path = PurePath.joinpath(context.include_files_directory, name1)
```

Comment:
Use `Path` instead of `PurePath`. We create `Path` objects below anyway.

Reply:
Changed.